src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -389,115 +438,129 @@ function ReportActionCompose({
                     style={isComposerFullSize ? styles.chatItemFullComposeRow : {}}
                     contentContainerStyle={isComposerFullSize ? styles.flex1 : {}}
                 >
-                    <View
-                        ref={containerRef}
-                        style={[
-                            shouldUseFocusedColor ? styles.chatItemComposeBoxFocusedColor : styles.chatItemComposeBoxColor,
-                            styles.flexRow,
-                            styles.chatItemComposeBox,
-                            isComposerFullSize && styles.chatItemFullComposeBox,
-                            hasExceededMaxCommentLength && styles.borderColorDanger,
-                        ]}
+                    <EducationalTooltip
+                        shouldRender={!shouldHideEducationalTooltip && shouldShowEducationalTooltip}
+                        renderTooltipContent={renderWorkspaceChatTooltip}
+                        shouldUseOverlay
+                        onPressOverlay={() => User.dismissWorkspaceTooltip()}
+                        anchorAlignment={{
+                            horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
+                            vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                        }}
+                        wrapperStyle={styles.reportActionComposeTooltipWrapper}
+                        shiftHorizontal={variables.composerTooltipShiftHorizontal}
+                        shiftVertical={variables.composerTooltipShiftVertical}
```

Comment:
On mobile safari, the viewport is scrolled when the keyboard is shown. We should also add the `topOffset` here to avoid [this bug](https://github.com/Expensify/App/issues/53976).

More details: https://github.com/Expensify/App/issues/53976#issuecomment-2537842083